rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/PropertyKeyMapping.java

```diff
@@ -0,0 +1,17 @@
+package org.eclipse.sw360.rest.resourceserver.core;
```

Comment:
add license header

Reply:
_to understand the usage_
This file is only used in case of changing thrift field names with jackson by using `@JsonProperty`?

Reply:
The problem with the license header is fixed now and I've added the documentation to the `components.adoc`

Reply:
Yes, the PropertyKeyMapping-Class only contains the logic and keys to map between thrift and jackson field names.